cpp/ql/src/semmle/code/cpp/Enum.qll

```diff
@@ -2,7 +2,13 @@ import semmle.code.cpp.Type
 private import semmle.code.cpp.internal.ResolveClass
 
 /**
- * A C/C++ enum [N4140 7.2].
+ * A C/C++ enum [N4140 7.2]. For example, the type `MyEnum` in:
+ * ```
+ * enum MyEnum
```

Comment:
It doesn't have to be in the example, but I think it's worth stating that this class includes C++11 scoped enums (`ScopedEnum` in QL).

Reply:
Done.